.cirrus.yml

```diff
@@ -81,32 +81,38 @@ gcp_docker_auth_win: &gcp_docker_auth_win
 
 task:
   matrix:
-    - name: freebsd-13
-      env:
-        PACKERFILE: packer/freebsd.pkr.hcl
-
-    - env:
-        PACKERFILE: packer/linux_debian.pkr.hcl
-
+    - skip: $CIRRUS_LAST_GREEN_CHANGE != '' && $CIRRUS_CRON != 'regular-rebuild' && !changesInclude('.cirrus.yml', 'docker/linux_debian_packer', 'scripts/linux_debian_*', $PACKERFILE)
       matrix:
-        - name: bullseye
-        - name: sid
-        - name: sid-newkernel
-        - name: sid-newkernel-uring
-
-    - env:
-        PACKERFILE: packer/windows.pkr.hcl
-
+        - name: freebsd-13
+          env:
+            PACKERFILE: packer/freebsd.pkr.hcl
+            SCRIPTS: scripts/linux_debian_*
```

Comment:
Freebsd uses linux_debian_*?

And I don't think you actually use SCRIPTS anywhere?